resources/user-guides/restore_from_keys.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: static_page
+title: "Restoring a wallet from private keys"
+---
+
+
+
+Restoring a wallet from private keys (via the command line in this instance) is pretty simple. If you have the necessary information, with this guide you can completely restore your wallet. Note, you do NOT have to have your password to restore from keys.
+
+You need to have 3 peices of data from your wallet, or your .keys file which holds this info and the password to decrypt it. The 3 wallet components that you need are:
+
+1. **Address**
+2. **Secret Spendkey**
+3. **Secret Viewkey**
+
+
+Then run the wallet command as such:
+
+`./monero-wallet-cli New_Wallet_Name.abc`
+
+Then, in turn you'll be asked for the Address, the spendkey, the viewkey, and finally the new password for the re-generated wallet.
+
+Running this command correctly with the correct parameters will re-generate your wallet files for you and allow you to set a new password.
```

Comment:
> correctly with the correct parameters

Running it correctly implies with the correct parameters. Maybe should be: `Running this command with the correct parameters will re-generate`